### PR TITLE
If "dad.curCharacter" is same as "gf.curCharacter"

### PR DESCRIPTION
**Pull request by rodney528**
Basically, ya know-how in the tutorial gf is technically the dad? Well, this makes it (I hope) so it can work with anyone other than just gf.

### CODE REVIEW

**Review by DabbinDude316B — APPROVED** [review]

**Review by TheKitBoi — APPROVED** [review]

**Review by MemeHoovy — APPROVED** [review]

**Comment by ImaginationSuperHero528** [comment]
> change the "dad.curCharacter.startsWith(gf.curCharacter)" to "dad.curCharacter == gf.curCharacter"
> for every if statement.

That would definitely work better.

**Review by SoLuzid — APPROVED** [review]

File: source/PlayState.hx [hunk 1]
```diff
@@ -3698,11 +3698,11 @@ class PlayState extends MusicBeatState
 								addCharacterToList(value2, charType);
 							}
 
-							var wasGf:Bool = dad.curCharacter.startsWith(gf.curCharacter);
+							var wasGf:Bool = dad.curCharacter == gf.curCharacter;
 							var lastAlpha:Float = dad.alpha;
 							dad.alpha = 0.00001;
 							dad = dadMap.get(value2);
-							if(!dad.curCharacter.startsWith(gf.curCharacter)) {
+							if(!dad.curCharacter == gf.curCharacter) {
```
Comment (kingyomoma) [inline source/PlayState.hx:3705]: change this to `if (dad.curCharacter != gf.curCharacter)`

**Review by iym1337 — CHANGES_REQUESTED** [review]

File: source/PlayState.hx (hunk 1 above)
Comment (ImaginationSuperHero528) [reply source/PlayState.hx:3705]: Ok I'll get to it soon.

Comment (ImaginationSuperHero528) [reply source/PlayState.hx:3705]: At school rn

**Comment by ImaginationSuperHero528** [comment]
:(

**Pull request closed without merging**